lib/smart_proxy_openscap/storage_fs.rb

```diff
@@ -45,8 +50,14 @@ def get_arf_file(digest)
 
     def get_path(digest)
       full_path = @path + digest
-      raise FileNotFound, "Can't find path #{full_path}" if !File.file?(full_path) || File.zero?(full_path)
+      if !File.file?(full_path) || File.zero?(full_path)
```

Comment:
When could the file be empty?

Reply:
Just keeping the condition that was already there. If you wish I could remove it, I just thought that I would change only what is necessary to not accidentally break anything.